### PR TITLE
Added API Calls for Image Gallery

### PR DESCRIPTION
**Pull request by claire123c**
- Added API call for styles and product details in Overview.jsx
- Added one test for styles api call, but needs to improved later on

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ app.get('/products/:product_id', (req, res) => {
       res.status(200).send(response.data);
     })
     .catch(error => {
+      console.log(error);
```
Comment (johnamielyasis) [inline server/index.js:39]: The only note I have is this this console.log but other than that lgtm

File: client/src/components/overview/Overview.jsx [hunk 2]
```diff
@@ -1,12 +1,37 @@
...
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
+import Gallery from './ImgGallery/Gallery.jsx';
 import sampleData from './sampleData.js';
 
 function Overview() {
+  const [productNum] = useState('25173');
+  const [productData, setProductData] = useState(sampleData.results);
+  const getProductDeets = () => {
```
Comment (Pakiavelli) [inline client/src/components/overview/Overview.jsx:10]: cute function name

**Review by johnamielyasis — APPROVED** [review]
Please see comment, but otherwise lgtm

**Review by Pakiavelli — APPROVED** [review]
Looks good to me, enjoying the descriptive variable naming

File: server/index.js (hunk 1 above)
Comment (claire123c) [reply server/index.js:39]: ty! will change that

**Pull request merged**